souschef.py

```diff
@@ -25,19 +28,133 @@
+		https://github.com/learningequality/sushi-chef-better-world-ed/pull/1
+		Thus, the database appears as:
+		+-------------------+------------+--------------------+---------------+-------+-------------+-----------+
+		| Grade Level Range | Math Topic | Specific Objective | Written Story | Video | Lesson Plan | BWE Topic |
+		+-------------------+------------+--------------------+---------------+-------+-------------+-----------+
+		with the names of the columns appearing in the first row, meaning all records begin at row 2.
+		At this time (11/10/2017), the "BWE Topic" column is not utilized. The columns are zero indexed,
+		so column 0 corresponds to "Grade Level Range", and column 6 corresponds to "BWE Topic".
+
+        Args: writer (Datawriter): class that writes data to folder/csv structure
+		Returns: None
+    """
 
+	count = 0 # Temporary, to verify correct output
+
+	with open(filename, 'r') as csvfile:
+		# Creating csv reader object
+		csvreader = csv.reader(csvfile)
+
+		# Extracting each data row one by one
+		for row in csvreader:
+			if count == 0: # Skip the headers
+				count += 1
+				continue
+			if count == 2: # Temporary, to ensure code works on the first few rows
+				break
+
+			# Some folders are named as hyperlinks
+			gradeLevel = row[0]
+			if gradeLevel == "":
+				gradeLevel = "Other"
+
+			# Setting the math topic and specific objective, as the database is
+			# zero indexed, row[1] will correspond to the current row, column 1,
+			# which is the title of the math topic. The specific objective is similar,
+			# except it takes in the specific objective, at column 2.
+			# Actual input will be sanitized with the try/except blocks immediately following.
+			mathTopic = row[1]
+			specificObjective = row[2]
+
+			# There are records in which the math topic and specific objectives in the
+			# database are given as hyperlinks, with the structure:
+			# "=HYPERLINK("https://www.khanacademy.org/math/cc-fourth-grade-math/cc-4th-fractions-topic","Fractions")"
+			# Thus, regular expressions are used to take in the correct title. See note 1 above.
+			# TODO: Ask if there's anything we should be doing with the url.
+			try:
+				match = re.findall(r'\"(.+?)\"', row[1])
+				mathTopic = match[1].replace('/', '|')
+			except:
+				mathTopic = row[1].replace('/', '|')
+			try:
+				match = re.findall(r'\"(.+?)\"', row[2])
+				specificObjective = match[1].replace('/', '|')
+			except:
+				specificObjective = row[2].replace('/', '|')
+
+			# Set the path and add folders
+			PATH.set(gradeLevel, mathTopic, specificObjective)
+			writer.add_folder(str(PATH), specificObjective)
+
+			# Written story (3th column, zero indexed)
+			try:
+				matches = re.findall(r'\"(.+?)\"', row[3])
+				title = matches[1].replace('/', '|')
+				print ("Adding written story: " + matches[1])
+				writer.add_file(str(PATH), title, matches[0], ext=".pdf", license=licenses.CC_BY, copyright_holder="betterworlded")
```

Comment:
Move copyright_holder and license to be global variable, copyright holder should be "Better World Education"